core/arch/arm/kernel/boot.c

```diff
@@ -75,7 +75,9 @@ DECLARE_KEEP_PAGER(sem_cpu_sync);
 #ifdef CFG_DT
 struct dt_descriptor {
 	void *blob;
+#ifdef CFG_EXTERNAL_DTB_OVERLAY
```

Comment:
Change seems unrelated. Fixes a build issue?

Reply:
No, it doesn't fix a build issue. I can move this into a separate patch if that's preferred.